src/index.css

```diff
@@ -1,16 +1,16 @@
 body {
-  font-family: "SangBleu OG Serif Light Regular";
+  font-family: "helvetica neue";
   text-decoration: none;
+  color: #f4f4f6;
 }
 
-a {
-  text-decoration: none;
+button {
   cursor: pointer;
 }
 
-button {
-  font-family: "SangBleu OG Serif Light Regular";
-  cursor: pointer;
+a {
+  text-decoration: none;
+  color: #f4f4f6;
```

Comment:
Maybe we should make a ticket for this whole file of css to become a styled-components `css` block which gets imported in `index.tsx` or something

Reply:
Sounds legit - #74